core/event.js

```diff
@@ -96,6 +96,15 @@ if ( !CKEDITOR.event ) {
 			/**
 			 * Registers a listener to a specific event in the current object.
 			 *
+			 * **Note**: CKEditor's event system has a limitation that one function cannot be used as listener for the same event more than once.
+			 * Hence, wrapper functions:
+			 *
+			 * ```javascript
+			 * this._listeners.push( win.on( 'scroll', function() {
```

Comment:
I would rework this example to be simpler and to show attaching to multiple events. It should use the same convention (`someObject.on...`) as rest of code samples here so something like:

```javascript
function listener( evt ) { ... };

someObject.on( 'someEvent', function() {
    listener();
} );

someObject( 'anotherEvent', function( evt ) {
    listener( evt );
} );
```